dev_suites/dev_custom_results/custom_result_suite.rb

```diff
@@ -0,0 +1,119 @@
+module CustomResult
+  class PassingCustomResultGroup < Inferno::TestGroup
+    id :passing_custom_result_group
+    title 'Passing Group'
+    description 'Criteria: Passes if test 1 passes, or test 2 and test 3 pass'
+
+    customize_passing_result do |results|
```

Comment:
I don't think this is quite there yet.

- `customize_passing_result` is a confusing name.
- This doesn't allow groups to do all of the things with results that tests can do, like add messages and summary messages
- It would be great if assertions and the rest of the test dsl could be used here

What if groups had the option of containing a `run` block like tests do, and the results were made available via an `attr_accessor`?